### DISCUSSION TITLE
ValueError: format number 1 of "decimal128(10, 0)" is not recognized

### DISCUSSION THREAD

**Discussion by dclong** [vaexio/vaex#1605] [Q&A]
Aggregating (`max`) on a column of the type `decimal128(10, 0)` raised the following error message. 
 ```
ValueError: format number 1 of "decimal128(10, 0)" is not recognized
```
It is quite confusing to me what the error message means. Does it mean that the format `decimal128(10,0)` is not supported in aggregations?

**Comment by dclong** [comment]
Also, adding 1 to the column `df.x + 1` throws the following error message. 
```
UFuncTypeError: ufunc 'add' did not contain a loop with signature matching types (dtype('S16'), dtype('S16')) -> dtype('S16')
```
How can I fix those issues? Should I cast decimal128(10,0) to a supported type?

**Comment by JovanVeljanoski** [comment]
I also do not recognize this  type :)

Vaex supports numpy and arrow dtypes, as well as the python primitives. 

Maybe you can go through the vaex tutorial on the documentation to get a better sense of the usage?

**Comment by dclong** [comment]
The data is read in from a Parquet file (which is fetched to local from a Hadoop server). It is common for SQL-like languages to support `decimal(n,k)` types. 

I just wondering is the `decimal(n,k)` type not recognized by Vaex?

**Comment by JovanVeljanoski** [comment]
Yeah I see it as an arrow type indeed. But yeah, it is not supported currently by vaex. I can look into it when I have time, you are very welcome to do a PR on this yourself! 

Thanks for the report, was not aware the decimal types were part of arrow!
